command/harness/delegate/delegate.go

```diff
@@ -112,7 +112,7 @@ func (c *delegateCommand) run(*kingpin.ParseContext) error {
 
 	instanceStore := database.ProvideInstanceStore(db)
 	c.stageOwnerStore = database.NewStageOwnerStore(db)
-	c.poolManager = drivers.New(ctx, instanceStore, c.env.Settings.LiteEnginePath, c.env.Runner.Name)
+	c.poolManager = drivers.New(ctx, instanceStore, c.env.Settings.LiteEnginePath, c.env.Runner.Name, false)
```

Comment:
we need to set it to true

Reply:
Made this typo when copying from drone-runner-aws repo to its fork in my local. I'll fix it.